Templating/Helper/LocaleHelper.php

```diff
@@ -43,7 +43,8 @@ public function language($code, $locale = null)
     {
         $codes = explode('_', $code);
 
-        $name = Intl::getLanguageBundle()->getLanguageName($codes[0], isset($codes[1]) ? $code[1] : null, $locale ?: $this->localeDetector->getLocale());
+        $name = Intl::getLanguageBundle()->getLanguageName($codes[0], isset($codes[1]) ? $codes[1] : null, 
```

Comment:
Someone should make another PR to solve this with dependency injection. I'm saying another PR because it will most likely be a BC-break.